src/toast/close-button/_toast-close-button.scss

```diff
@@ -0,0 +1,5 @@
+.toast-close-button {
+  --toast-close-button-z-index: 3;
+  
+  z-index: var(--toast-close-button-z-index);
+}
```

Comment:
No need for the css variable. I think we should rethink the use of extensive css variables. Many of them within toolkit are actually redundant because we can easily override the styles within projects now with CSS.

Reply:
Maybe we can think we need which variables and then we can remove.